assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -542,10 +542,8 @@ che.singleport.wildcard_domain.ipless=false
 
 # Docker image of Che plugin broker app that resolves workspace tooling configuration and copies
 # plugins dependencies to a workspace
-che.workspace.plugin_broker.image=eclipse/che-plugin-broker:v0.14.0
-che.workspace.plugin_broker.theia.image=eclipse/che-theia-plugin-broker:v0.14.0
 che.workspace.plugin_broker.init.image=eclipse/che-init-plugin-broker:v0.14.0
```

Comment:
Should not it be updated to `v0.15.0`?

Reply:
There are no changes in the init broker, so no need to update it. Just reducing not needed changes.